### PR TITLE
feat(portal): initial vortex portal

### PR DESCRIPTION
**Pull request by yangpangpangya**
## Description

init the source code in vortex-app/vortex-app-portal

### CODE REVIEW

File: vortex-app/vortex-app-portal/.env [hunk 1]
```diff
@@ -1 +1,2 @@
-VITE_BASE_API=http://localhost:8001
\ No newline at end of file
+
+CONSOLE_MODULES="examlpe@http://localhost:3003/remoteEntry.js"
```
Comment (DaveXiong) [inline vortex-app/vortex-app-portal/.env:2]: better replace `CONSOLE_MODULES` with `VOTEX_MODULES`

File: vortex-app/vortex-app-portal/LICENSE [hunk 2]
```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```
Comment (DaveXiong) [inline vortex-app/vortex-app-portal/LICENSE:1]: Pls remove this LICENSE file, the LICENSE is in the base of this repo

File: vortex-app/vortex-app-portal/README.md [hunk 3]
```diff
@@ -1,30 +1,6 @@
...
-}
-```
-
-- Replace `plugin:@typescript-eslint/recommended` to `plugin:@typescript-eslint/recommended-type-checked` or `plugin:@typescript-eslint/strict-type-checked`
-- Optionally add `plugin:@typescript-eslint/stylistic-type-checked`
-- Install [eslint-plugin-react](https://github.com/jsx-eslint/eslint-plugin-react) and add `plugin:react/recommended` & `plugin:react/jsx-runtime` to the `extends` list
+# vortex-portal
+Vortex FE 
+# vortex repository list
+- module tempalte: https://github.com/Console-Core/vortex-module-template
```
Comment (DaveXiong) [inline vortex-app/vortex-app-portal/README.md:4]: Pls update the README

File: vortex-app/vortex-app-portal/package.json [hunk 4]
```diff
@@ -1,66 +1,93 @@
 {
-  "name": "vortex-app-portal",
-  "private": true,
-  "version": "0.0.0",
-  "type": "module",
+  "name": "vortex-portal",
```
Comment (DaveXiong) [inline vortex-app/vortex-app-portal/package.json:2]: name should be `vortex-app-portal`

File: vortex-app/vortex-app-portal/package.json [hunk 5]
```diff
@@ -1,66 +1,93 @@
...
+  "version": "1.0.0",
```
Comment (DaveXiong) [inline vortex-app/vortex-app-portal/package.json:3]: version should be `0.0.0`

File: vortex-app/vortex-app-portal/src/assets/default-logo.svg
Comment (DaveXiong) [inline vortex-app/vortex-app-portal/src/assets/default-logo.svg:1]: Pls change this logo to the correct one

Comment (yangpangpangya) [reply vortex-app/vortex-app-portal/src/assets/default-logo.svg:1]: this logo from UI, It is just a placeholder for now, we will replace it with the environment variable later

Comment (yangpangpangya) [reply vortex-app/vortex-app-portal/src/assets/default-logo.svg:1]: have  been removed

**Review by DaveXiong — APPROVED** [review]
LGTM

**Pull request merged**